spec/services/section/create_service_spec.rb

```diff
@@ -0,0 +1,96 @@
+require "spec_helper"
+
+RSpec.describe Section::CreateService do
+  let(:user) { User.gds_editor }
+  let(:manual) { Manual.new(title: 'manual-title') }
+  let(:section_attributes) { double(:section_attributes) }
+  let(:publishing_api_adapter) { double(:publishing_api) }
+  let(:new_section) {
+    Section.new(manual: manual, uuid: 'uuid', editions: [])
+  }
+
+  subject do
+    described_class.new(
+      user: user,
+      manual_id: manual.id,
+      attributes: section_attributes
+    )
+  end
+
+  before do
+    allow(Manual)
+      .to receive(:find).with(manual.id, user)
+      .and_return(manual)
+    allow(manual)
+      .to receive(:build_section).with(section_attributes)
+      .and_return(new_section)
```

Comment:
I tried to use the real `Manual#build_section` method initially but ran into problems when trying to test the call to `Adapters.publishing.save_section` with the section that's returned from `build_section`. I'm going to leave this as it is at the moment and see whether I come up with anything better while working on the rest of the services.